corehq/apps/reports/standard/deployments.py

```diff
@@ -387,33 +386,49 @@ def total_records(self):
     @property
     def rows(self):
         if self.warehouse:
-            mobile_user_and_group_slugs = set(
-                # Cater for old ReportConfigs
-                self.request.GET.getlist('location_restricted_mobile_worker') +
-                self.request.GET.getlist(ExpandedMobileWorkerFilter.slug)
-            )
-            users = ExpandedMobileWorkerFilter.user_es_query(
-                self.domain,
-                mobile_user_and_group_slugs,
-                self.request.couch_user,
-            ).values_list('_id', flat=True)
+            user_ids = self.get_user_ids()
             sort_clause = self.get_sql_sort()
             rows = ApplicationStatusFact.objects.filter(
-                user_dim__user_id__in=users
+                user_dim__user_id__in=user_ids
```

Comment:
not related to this PR but I thought half the point of moving to the warehouse was so that we didn't have to do this but could instead join on the user fact table with appropriate filters.

Reply:
Yes, true. I wanted to make that a separate PR, this PR being just about making exports use similar style as the non-export version/